frontend/src/utils/protobuf-reflection.test.ts

```diff
@@ -42,7 +43,7 @@ describe('Protobuf Field Reflection', () => {
   test('should detect field behaviors for KnowledgeBaseUpdate', () => {
     debugFieldBehaviors(KnowledgeBaseUpdateSchema);
 
-    const _metadata = getMessageFieldMetadata(KnowledgeBaseUpdateSchema);
+    getMessageFieldMetadata(KnowledgeBaseUpdateSchema);
```

Comment:
[blocker]: is this function doing side-effects, if so I would prefer to refactor that if not this is a void invocation.